qiita_ware/ebi.py

```diff
@@ -704,7 +704,7 @@ def write_xml_file(self, element, fp):
         fp : str
             The filepath to which the XML will be written
         """
-        create_dir(self.xml_dir)
+        makedirs(self.xml_dir)
         ET.ElementTree(element).write(fp, encoding='UTF-8')
```

Comment:
This wouldn't be UTF-8 by default?

Reply:
Not sure ... but I rather don't change anything I'm not sure about yet (this line didn't change )